### ISSUE TITLE
According to the code description in your README, I am unable to successfully run the program.

### ISSUE THREAD

**Issue by tianyouzhao789** [GaochangWu/FMF-Benchmark#3]
**Hello Author,**

I have a question regarding the preprocessing of your open-source dataset for successfully executing the following command:
`python tools/test_cls.py --cfg './configs/CfgForViCu_Cls.yaml' DATA.PATH_TO_DATA_DIR './FMFBenchmarkV1/pixel-level' MODEL.PRETRAINED './state_dict_vicu_cls.pth' DATA.SAMPLING_RATE 18`

I noticed that if I only place the three `.mat` files in the data directory and run the above command, it throws errors such as `test.json` file not found and `set3.csv` file not found. Could you kindly provide some guidance on how to handle the dataset?

Additionally, could you please check if my current approach to processing the dataset is correct? 
The `.mat` files contain four parts: video, current, label, and train_test_index. My idea is to extract these four parts separately, save the video as individual `.mat` files, and store the current data as `.csv` files.

Looking forward to your advice. Thank you!

**Comment by tianyouzhao789** [comment]
Could you please let me know what the file structure of the `datasetPath/FMF-Benchmark/pixel-level` folder looks like and what contents are stored inside it?

**Comment by studyfight** [comment]
Excuse me, has your problem been solved?

**Comment by tianyouzhao789** [comment]
> Excuse me, has your problem been solved?

no

**Comment by tianyouzhao789** [comment]
I rewrote his code.

> Excuse me, has your problem been solved?

**Comment by studyfight** [comment]
The other two files exist, but the file named "datasetPath/FMF-Benchmark/pixel-level" is missing. Regarding this, how should it be modified?

**Comment by tianyouzhao789** [comment]
> The other two files exist, but the file named "datasetPath/FMF-Benchmark/pixel-level" is missing. Regarding this, how should it be modified?另外两个文件存在，但名为“datasetPath/FMF-Benchmark/pixel-level”的文件缺失。关于这一点，应该如何修改？
Timestamp of where you came from

**Comment by studyfight** [comment]
I'm a bit confused about what you mean. What is "Timestamp"? Perhaps you're asking where I'm from?

**Comment by tianyouzhao789** [comment]
> I'm a bit confused about what you mean. What is "Timestamp"? Perhaps you're asking where I'm from?


The dataset published by the author does not include the element of a timestamp, but the code relies on timestamps to align the current data with the video data. In the code released by the author, timestamps are of critical importance.

**Comment by studyfight** [comment]
> > I'm a bit confused about what you mean. What is "Timestamp"? Perhaps you're asking where I'm from?我有点搞不懂你的意思。“时间戳”是什么？也许你是在问我来自哪里？
> 
> The dataset published by the author does not include the element of a timestamp, but the code relies on timestamps to align the current data with the video data. In the code released by the author, timestamps are of critical importance.作者公布的数据集中没有包含时间戳这个元素，但是代码中依赖时间戳来对齐当前数据和视频数据，在作者公布的代码中，时间戳是至关重要的。

So, do we need to find a new dataset? A dataset with timestamps, and then put it into the corresponding directory of the author?

**Comment by studyfight** [comment]
Did you find a new dataset or modify the part related to the timestamp in the code?

**Comment by GaochangWu** [comment]
Hi! I've just uploaded a new version of the dataset, which now includes diverse sampling rates along with a "Timestamp" for each data point. You can download it from the shared Baidu Disk. I’ll also update the ReadMe file shortly. Please give me a moment to complete the updates.

**Comment by studyfight** [comment]
> Hi! I've just uploaded a new version of the dataset, which now includes diverse sampling rates along with a "Timestamp" for each data point. You can download it from the shared Baidu Disk. I’ll also update the ReadMe file shortly. Please give me a moment to complete the updates.嗨！我刚刚上传了新版本的数据集，现在它包括不同的采样率以及每个数据点的“时间戳”。您可以从共享的百度网盘下载它。我也会很快更新 ReadMe 文件。请给我一点时间来完成更新。

thank you

**Comment by Chenghuijie0204** [comment]
我有使用你修改后的代码，但是同样会出现yaml文件找不到的错误，请问你是怎么处理这个的？
